### DISCUSSION TITLE
Project Communication

### DISCUSSION THREAD

**Discussion by auronandace** [GyulyVGC/sniffnet#6] [Ideas]
How do you feel about setting up an IRC channel on libera.chat or a Matrix channel for the project for a way for interactive discussion?

**Comment by GyulyVGC** [comment]
Hi! First of all thank you for the suggestion, I appreciate it.
However, unluckily I’m busy with many academic projects and the time I plan
to dedicate to Sniffnet is limited.
I think GitHub is enough for my goals.

**Comment by GyulyVGC** [comment]
Update: I've just setup a [Telegram Channel](https://t.me/sniffnet) for Sniffnet.
Despite it's not an interactive communication channel, I believe this is the start of a sort of community around the project!

An RSS feed will follow soon!
